rpc/client.go

```diff
@@ -16,12 +21,19 @@ import (
 )
 
 // UploadReportOptions contains all the required options for uploading a report
-// to both Cedar and the Data Pipes service.
+// to both Cedar and SPS.
+
 type UploadReportOptions struct {
 	Report          *poplar.Report
-	ClientConn      *grpc.ClientConn
+	CedarClientConn *grpc.ClientConn
+	SPSClientConn   *http.Client
```

Comment:
I would call this `HTTPClient` and make it optional (you can just use the HTTP client pool in the `utility` package if not provided).